apps/shared/models/pull-request.ts

```diff
@@ -0,0 +1,88 @@
+import {PullRequest} from '@octokit/webhooks-types';
+import {GithubBaseModel, GithubHelperFunctions} from './base';
+import {GithubLabel} from './label';
+import {GithubMilestone} from './milestone';
+import {GithubTeam} from './team';
+import {GithubUser, isUser} from './user';
+
+interface FirestorePullRequest {
+  owner: string;
+  repo: string;
+  node: string;
+  state: PullRequest['state'];
+  authorAssociation: PullRequest['author_association'];
+  body: string;
+  changedFiles: number;
+  closedAt: string | null;
+  commits: number;
+  createdAt: string;
+  draft: boolean;
+  labels: string[];
```

Comment:
Can we denote these as actual references somehow? using e.g. branded strings? I think it's easy to confuse this with an actual label name right? same for other foreign "values/keys"